ext/src/ruby_api/module.rs

```diff
@@ -106,6 +109,24 @@ impl Module {
     pub fn get(&self) -> &ModuleImpl {
         &self.inner
     }
+
+    /// @yard
+    /// Returns the list of imports that this Module has and must be satisfied.
+    /// @return [Array<Hash>] An array of hashes containing import information
+    pub fn imports(&self) -> Result<RArray, Error> {
+        let module = self.get();
+        let imports = module.imports();
+
+        let result = RArray::new();
+        for import in imports {
+            let hash = RHash::new();
```

Comment:
The exports' length is known, I believe here we can use `RArray::with_capacity(imports.len())` to minimize the cost of reallocations. We can do the same with the usage of `RHash`.